src/Views/Main/Directors/Directors.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import { fetchDirectors } from '../../../services/directors';
+import Select from '../../../components/Select';
+
+
+export default function Directors() {
+  const [directors, setDirectors] = useState ([]);
+  const [name, setDirect] = useState ('All');
+
+  useEffect(() => {
+    const fetch = async () => {
+      try {
+        const resp = await fetchDirectors(name);
+        setDirectors(resp);
+      } catch (e) {
+        alert(e.message);
+      }
+
+    };
+    fetch();
+
+  }, [name]);
+        
+    
+  return (
+    <div className='director'>
+      <Select setDirect={setDirect} />
+      {directors.map((director) => (
+        <div key={director.id}>
```

Comment:
Great working utilizing the .map and targeting multiple properties to display on the page